### PR TITLE
Change timeout for email resend to 10 Minutes

### PR DESCRIPTION
**Pull request by einhornimmond**
### Issues

- None

### Todo

- [X] None

### CODE REVIEW

**Comment by einhornimmond** [comment]
Sorry, another branch is slipped into this PR also.
It's add the basis for a new Test ( Test Json Create Transaction call ) and handle an error which occure not really often.

File: login_server/src/cpp/lib/JsonRequest.cpp [hunk 1]
```diff
@@ -36,6 +36,10 @@ JsonRequestReturn JsonRequest::request(const char* methodName, const Poco::JSON:
 	// send post request via https
 	// 443 = HTTPS Default
 	// TODO: adding port into ServerConfig
+	if (mServerHost.empty() || !mServerPort) {
+		addError(new Error(functionName, "server host or server port not given"));
+		return JSON_REQUEST_PARAMETER_ERROR;
+	}
```
Comment (Mogge) [inline login_server/src/cpp/lib/JsonRequest.cpp:42]: What has this change to do with the timeout?

File: login_server/src/cpp/lib/JsonRequest.h [hunk 2]
```diff
@@ -20,6 +20,7 @@ enum JsonRequestReturn
 	JSON_REQUEST_RETURN_OK,
 	JSON_REQUEST_RETURN_PARSE_ERROR,
 	JSON_REQUEST_RETURN_ERROR,
+	JSON_REQUEST_PARAMETER_ERROR,
```
Comment (Mogge) [inline login_server/src/cpp/lib/JsonRequest.h:23]: Same here. Timeout?

File: login_server/src/cpp/test/JSONInterface/TestJsonCreateTransaction.cpp [hunk 3]
```diff
@@ -0,0 +1,51 @@
+#include "TestJsonCreateTransaction.h"
```
Comment (Mogge) [inline login_server/src/cpp/test/JSONInterface/TestJsonCreateTransaction.cpp:1]: What have the transactions to do with the timeout?

File: login_server/src/cpp/test/JSONInterface/TestJsonCreateTransaction.h [hunk 4]
```diff
@@ -0,0 +1,23 @@
+#ifndef __GRADIDO_LOGIN_SERVER_TEST_JSON_INTERFACE_TEST_JSON_CREATE_TRANSACTION_H
```
Comment (Mogge) [inline login_server/src/cpp/test/JSONInterface/TestJsonCreateTransaction.h:1]: Same

**Review by Mogge — APPROVED** [review]
The changes for the timeout are ok. What are the other changes for? It is really confusing reviewing code when you do not know what it is meant to do.

**Pull request merged**